v3/src/components/graph/models/data-configuration-model.ts

```diff
@@ -31,7 +36,7 @@ export const DataConfigurationModel = types
     dataset: undefined as IDataSet | undefined,
     actionHandlerDisposer: undefined as (() => void) | undefined,
     filteredCases: undefined as FilteredCases | undefined,
-    handlers: new Map<string,(actionCall: ISerializedActionCall) => void>()
+    handlers: new Map<string, (actionCall: ISerializedActionCall) => void>()
```

Comment:
```suggestion
    handlers: new Map<string,(actionCall: ISerializedActionCall) => void>()
```
The unexpected white space warning is technically a parsing error on the part of eslint -- the parenthesis at issue here isn't actually part of a function call -- but the simple fix is to eliminate the space. I turned this warning on because it gives a more informative error message when there are lines of code that begin with a parenthesis and therefore require an additional semicolon.